neighbors/src/components/about/aboutLayout.css

```diff
@@ -0,0 +1,84 @@
+.container {
+  position: relative;
+  height: 100vh;
+  width: 70%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 20px 20px;
+  justify-content: center;
+}
+
+h1 {
+  text-align: center;
+  font-size: 2em;
+}
+p {
+  text-align: left;
+  padding: 10px 20px;
+  font-size: 1.5em;
+}
+.left {
+  width: 100%;
+  margin: 0 15px;
+}
+
+.icon {
+  font-size: 10em;
+  color: #acacac;
+}
+
+@media screen and (max-width: 1400px) {
+  .bg-gradient {
+    background-size: 200%;
+  }
+}
+
+@media screen and (max-width: 900px) {
+  .left {
+    width: 100%;
+    margin: 0;
+  }
+  .post-box {
+    padding-top: 50px;
+    grid-template-columns: 1fr;
+  }
+  p {
+    text-align: left;
+  }
+}
+
+@media screen and (max-width: 880px) {
+  .contact-box {
+    grid-template-columns: 1fr;
+  }
+  .left {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+  }
+  .post-box {
+    padding-top: 50px;
+    grid-template-columns: 1fr;
+  }
+  p {
+    text-align: left;
+  }
+}
+
+@media screen and (max-width: 390px) {
+  .icon {
+    width: 100px;
+    height: 100px;
+  }
+  .container {
+    padding: 20px;
+  }
+  .bg-gradient {
```

Comment:
jest juz w pierwszym media query wiec nic nie zmienia :) 
@media screen and (max-width: 1400px) {
  .bg-gradient {
    background-size: 200%;
  }
}